src/write.c

```diff
@@ -941,6 +941,35 @@ static avifCodecType avifEncoderGetCodecType(const avifEncoder * encoder)
     return avifCodecTypeFromChoice(encoder->codecChoice, AVIF_CODEC_FLAG_CAN_ENCODE);
 }
 
+// This function is called after every color frame is encoded. It returns AVIF_TRUE if a keyframe needs to be forced for the next
+// alpha plane that's encoded, AVIF_FALSE otherwise.
```

Comment:
alpha plane => alpha frame ?

that's encoded => to be encoded ?

Reply:
Done.